### DISCUSSION TITLE
What is the fastest way to display raw image data on the LCD for animation

### DISCUSSION THREAD

**Discussion by hacktronics** [Bodmer/TFT_eSPI#2260] [Q&A - General] [answered]
I am planning to use ST7735 based small 1.8inch LCD.
I can generate data in RGB pixels 8 bits also, so can have an array of pixels each pixels 3 bytes long.
What is the fastest way to display the raw data using the library, instead of setting individual pixels.
the goal is to display animations using an ESP32.

**Comment by Bodmer** [comment, marked as answer]
Fastest is 16 bit 565 format raw pixels stored in FLASH. See exampleTFT_Flash_Bitmap. You can then use pushImage().
